<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionConsumeDialog</class>
 <widget class="QDialog" name="ActionConsumeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoM Action Consume Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,2,1" columnstretch="1,0,0">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBoxAttacker">
     <property name="title">
      <string>Attackers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
      <item>
       <widget class="QListWidget" name="listWidgetTagOrGuild"/>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidgetAttackers">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButtonTokenUseAll">
        <property name="text">
         <string>Use all available tokens</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButtonTokenUseFixed">
          <property name="text">
           <string>Use fixed tokens</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxUseFixedTokens">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> tokens</string>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButtonTokenUseToLeft">
          <property name="text">
           <string>Use token until</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxUseUntil">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> token(s) left</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxLandActionLeft">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string> actions left</string>
        </property>
        <property name="prefix">
         <string>Stop consuming when </string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBoxDefender">
     <property name="title">
      <string>Defenders</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0" columnstretch="1,0,0">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxDefenderGuild"/>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxLandId">
        <property name="prefix">
         <string>Slot </string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonRefreshLand">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QListWidget" name="listWidgetDefenders"/>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBoxDefenderGuildOfficer"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBoxDefenderGuildMember"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEditLog"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonStart</tabstop>
  <tabstop>pushButtonStop</tabstop>
  <tabstop>radioButtonTokenUseFixed</tabstop>
  <tabstop>spinBoxUseFixedTokens</tabstop>
  <tabstop>radioButtonTokenUseToLeft</tabstop>
  <tabstop>spinBoxUseUntil</tabstop>
  <tabstop>spinBoxLandActionLeft</tabstop>
  <tabstop>comboBoxDefenderGuild</tabstop>
  <tabstop>spinBoxLandId</tabstop>
  <tabstop>pushButtonRefreshLand</tabstop>
  <tabstop>listWidgetDefenders</tabstop>
  <tabstop>listWidgetTagOrGuild</tabstop>
  <tabstop>tableWidgetAttackers</tabstop>
  <tabstop>plainTextEditLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
